_includes/faq/auto-generate-unique-ids_v1.1.html

```diff
@@ -0,0 +1,21 @@
+To auto-generate unique row IDs, use the [`SERIAL`](serial.html) data type, which is an alias for [`INT`](int.html) with the `unique_rowid()` [function](functions-and-operators.html#id-generation-functions) as the [default value](default-value.html):
```

Comment:
@jordanlewis, given what you said in another comment about the different use cases for `serial`, `bytes`, and `uuid`, do you think this faq needs to be updated? This language was written with @bdarnell. But at the very least, we may want to call out that `UUID` is provides good interoperability with applications that use UUIDs for identifier?